scripts/smg-html.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+
+import os
+import re
+import json
+
+# create directories if don't exist
+dirMain = "../output/smg-html"
+dirImg = "../output/smg-html/img"
+dirOutput = "../output"
+if not os.path.exists(dirMain):
+    os.makedirs(dirMain)
+if not os.path.exists(dirImg):
+    os.makedirs(dirImg)
+
+# clear main directory
+for fileName in os.listdir(dirMain):
+    filePath = os.path.join(dirMain, fileName)
+    try:
+        if os.path.isfile(filePath):
+            os.unlink(filePath)
```

Comment:
The whole thing could be simplified by using shutil.rmtree and then re-creating dirs again